src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -69,9 +68,8 @@ const RequestTable = ({
               firstInFuture = true;
             }
 
-            const isInLoadedRegion = loadedRegions
-              ? isTimeInRegions(row.original.point.time, loadedRegions.loaded)
-              : false;
+            const isInLoadedRegion =
+              focusWindow !== null && isPointInRegion(row.original.point.point, focusWindow);
```

Comment:
why not use `useIsPointWithinFocusWindow` here?

Reply:
You can't call a hook within a loop (`rows.map` in this case)

We could move the is-in-loaded-region check into the `RequestRow` component (and maybe I will) but on first pass, I was just trying to minimize changes.